cypress/e2e/smoke/Wallets/fiatWithdrawal.cy.js

```diff
@@ -58,17 +58,16 @@ describe('QATEST-98812 - Fiat withdrawal access iframe from email verification l
   it('should be able to access doughflow iframe', () => {
     cy.log('Access Fiat Withdrawal Iframe Through Email Link')
     cy.c_visitResponsive('/', 'large')
```

Comment:
do we not need a skip pass keys here?

```suggestion
    cy.c_visitResponsive('/', 'large')
    cy.c_skipPassKeysV2()
```

Reply:
@bilalmajid-deriv passkey is only visible in responsive view

Reply:
oh sorry my bad